pkg/agent/openflow/client.go

```diff
@@ -1130,3 +1144,33 @@ func (c *client) SendIGMPQueryPacketOut(
 	packetOutObj := packetOutBuilder.Done()
 	return c.bridge.SendPacketOut(packetOutObj)
 }
+
+func (c *client) InstallTrafficControlMarkFlows(name string, sourcePorts []uint32, targetPort uint32, direction v1alpha2.Direction, action v1alpha2.Action) error {
+	flows := c.featurePodConnectivity.trafficControlMarkFlows(sourcePorts, targetPort, direction, action)
+	cacheKey := fmt.Sprintf("tc_%s", name)
+	c.replayMutex.RLock()
+	defer c.replayMutex.RUnlock()
+	return c.addFlows(c.featurePodConnectivity.tcCachedFlows, cacheKey, flows)
```

Comment:
Would you consider update case? I suppose update should call `InstallTrafficControlMarkFlows`, then it should remove previous flows in same bundle?

Reply:
Updated with `c.modifyFlows`